server/service/integration_mdm_test.go

```diff
@@ -0,0 +1,182 @@
+package service
```

Comment:
with windows mdm coming, should we call this `integration_mdm_apple_test.go`? not sure, probably premature anyways.

Reply:
I thought about it a bit, but was hoping we could have a single suite for all mdm. In any case, as you mention we can make that decision when we add windows tests.